config/routes.rb

```diff
@@ -6,8 +6,10 @@
 
   namespace :api do
   	namespace :v1 do
-  		resources :shops, only: [:index]
+  		resources :shops, only: [:index, :show]
   	end
   end
-
+ 
+ get "*path", to: "homes#index" 
+ 
```

Comment:
We will no longer allow for "catch all" routing. Please designate _specific_ paths in your web app that you want to have that route to the "homes#index" action